expression/builtin_vectorized.go

```diff
@@ -67,6 +67,17 @@ func newBuffer(evalType types.EvalType, capacity int) (*chunk.Column, error) {
 	return nil, errors.Errorf("get column buffer for unsupported EvalType=%v", evalType)
 }
 
+// GetColumn allocates a column buffer with the specific eval type and capacity.
+// the allocator is not responsible for initializing the column, so please initialize it before using.
+func GetColumn(evalType types.EvalType, capacity int) (*chunk.Column, error) {
+	return globalColumnAllocator.get(evalType, capacity)
```

Comment:
Why not make `globalColumnAllocator` exported?

Reply:
Most of the vectorization builtin function will use the  ```globalColumnAllocator```. If we export the ```globalColumnAllocator```, many changes will happen. In order to reduce changes, we add ```GetColumn``` function to get a column in the buffer.

Reply:
``` go
expression/builtin_vectorized.go
48:var globalColumnAllocator = newLocalSliceBuffer(1024)

expression/expression.go
283:            buf, err := globalColumnAllocator.get(eType, n)
320:            globalColumnAllocator.put(buf)
```